scripts/build.sh

```diff
@@ -5,7 +5,6 @@ echo "Run with 'source ./scripts/build.sh [fed_size] [dir]"
 
 # allow for overriding arguments
 export FM_FED_SIZE=${1:-4}
-BASE_PORT=$((8173 + 10000))
```

Comment:
This variable was only used for DKG so moved it to be internal to that function.